app.js

```diff
@@ -52,6 +55,7 @@ searchForm.addEventListener('submit', (e) => {
     e.preventDefault();
     const formData = new FormData(searchForm);
     // > Part C: Call findCountries with name and continent from formData
+    findCountries(formData.get('name'), formData.get('continent'));
```

Comment:
this works fine, you could also store the name and continent in variables to increase readability